babel.config.js

```diff
@@ -5,5 +5,6 @@ module.exports = {
       { useBuiltIns: 'usage', corejs: '3' }
     ],
     '@babel/flow',
+    '@babel/typescript'
```

Comment:
I don't have a strong opinion either way. Once Flow is gone, I'm AOK with just `tsc`.

Reply:
My preference is to use the TS compiler for TS projects rather than babel.  

However, while there is a mixture of JS with Flow types and TS code, I reckon that babel is a better choice.  That is because babel can understand TS, Flow and JS simultaneously, while the TS compiler doesn't understand Flow (if you know of a way to precompile the Flow types out or some other voodoo let me know).  I've played around converting a few other files and the difficulties soon become apparent.  I might push up a couple more code changes that expose the issues.

I think in the long term, moving to a configuration that is the same/similar to what Jest itself uses would be the ideal.